Peli-Kiera/README.md

```diff
@@ -133,7 +157,7 @@ The pagination is presented with the current page and 3 neighboring pages on eit
       
 Images can be moved on page by adding the tag `#float-right`, `#float-left`, `#mid` and `#full` in your markdown.
 
-```markdown
+```bash
 ![Title]({static}/images/my_image-600x400.png#float-right "")
```

Comment:
Why the change to `bash` here, even though it's Markdown? Just curious.

Reply:
Hi @justinmayer ,

I had issues with that line on GitHub as well but it seems fine now. 
Short answer: it was just a work around and I am not sure on how to address it on my `pygment.css`

On my local machine I was getting this. 
![image](https://user-images.githubusercontent.com/33024269/109784419-2c2fce80-7c5f-11eb-85cc-866b5adfffde.png)

I got something similar for the JSON syntax highlighting as well yesterday. 
So I decided to remove the error class highlighting from `pygment.css`. That solved it. 

I appreciate the feedback and any other recommendations you may still have. Btw does it require a new PR ??